### DISCUSSION TITLE
Purpose of URI of dataset

### DISCUSSION THREAD

**Discussion by Max0u** [apache/airflow#35507] [Q&A] [answered]
What is the purpose of the URI in the dataset object? Is it just an identifier? Or there is some check of presence checks?

**Comment by potiuk** [comment, marked as answer]
That's it for now. Just identifier. Might change in the future. See https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-48+Data+Dependency+Management+and+Data+Driven+Scheduling for the scope of implemented feature of dataset and discussion on potential future changes.
